hw/ip_templates/alert_handler/doc/dv/index.md

```diff
@@ -15,7 +15,7 @@ title: "ALERT_HANDLER DV document"
 ## Current status
 * [Design & verification stage]({{< relref "hw" >}})
   * [HW development stages]({{< relref "doc/project/development_stages" >}})
-* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/results.html)
+* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/report.html)
```

Comment:
This should be: https://reports.opentitan.org/hw/top_earlgrey/ip_autogen/alert_handler/dv/latest/report.html

Reply:
Updated in [eb9d914](https://github.com/lowRISC/opentitan/compare/174738e3f56025c3f28bd6554b1c105c5c6513c3..eb9d9143f653b9d439d62694caf249d24cac12c5)